2019-11-20 16:48:55.465 - [Info] MITM Version: 1.26
2019-11-20 16:48:55.472 - [Info] Auto Access Enabled: true
2019-11-20 16:48:55.472 - [Debug] [Init] Auto Access Barrier: true
2019-11-20 16:48:55.497 - [Info] SSH man-in-the-middle server for 172.20.0.5 listening on 0.0.0.0:10104
2019-11-20 16:58:15.712 - [Debug] [Connection] Attacker connected: 119.29.128.126 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 16:58:17.227 - [Debug] [Auth] Attacker 119.29.128.126 trying to authenticate with "password"
2019-11-20 16:58:17.229 - [Debug] [Auto Access] Attacker: 119.29.128.126, Threshold: 1, Attempts: 1
2019-11-20 16:58:17.229 - [Debug] [Auto Access] Compromising the honeypot
2019-11-20 16:58:17.229 - [Debug] [Auto Access] Adding the following credentials: "vlasti:vlasti"
2019-11-20 16:58:17.557 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-20 16:58:17.557 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.5
2019-11-20 16:58:17.632 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-20 16:58:18.649 - [Debug] [Connection] Attacker connected: 119.29.128.126 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 16:58:20.216 - [Debug] [Auth] Attacker 119.29.128.126 trying to authenticate with "password"
2019-11-20 16:58:20.217 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-20 16:58:20.218 - [Debug] [Auth] Attacker: 119.29.128.126 has so far made 1 attempts. Remaining: 5 attempts
2019-11-20 16:58:20.753 - [Debug] [Connection] Attacker closed the connection
2019-11-20 16:58:21.215 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
